libs/designsystem/icon/src/icon-registry.service.spec.ts

```diff
@@ -37,18 +38,21 @@ describe('KirbyIconRegistryService', () => {
   });
 
   describe('getIcons', () => {
-    it('should return empty map by default', () => {
-      const expectedIcons = [];
+    it('should return defaultIcons by default', () => {
+      const expectedIcons = kirbyIconSettings.icons;
       expect(service.getIcons()).toEqual(expectedIcons);
     });
 
     it('should return registed icons', () => {
-      const icons = [
+      const defaultIcons = kirbyIconSettings.icons;
+      const customIcons = [
         { name: 'name1', svg: 'svg1' },
         { name: 'name2', svg: 'svg2' },
       ];
-      service.addIcons(icons);
-      expect(service.getIcons()).toEqual(icons);
+      const combinedIcons = defaultIcons.concat(customIcons);
```

Comment:
consider aligning concat and spread, e.g. reuse approach from line 78, so it goes like this:
`const expectedIcons = [...defaultIcons, { name: 'name1', svg: 'svg1' }, { name: 'name2', svg: 'svg2' } ];`